Source/Server.c

```diff
@@ -571,13 +571,14 @@ static void* serverConsole(void* arg)
     rl_clear_history();
 #endif
 
-    server_sleep(5); // seconds
+    StopServer();
```

Comment:
This is not needed at all. Like it was there before. Rebase your commits. Dont fix up in later ones :|